Schema 2.0.0

YambPix opened the pull request:
@olehkozak @dovdepixellot @roman-matviiv please make sure all of you review it before we publish.

## Schema Update Summary

**Version**: Updated from `1.0.0` → `2.0.0`

**Process:**
- Took the additions from the v2.0.0 added attributes specification
- Checked the published schema example from a real breakdown  
- Reviewed the current v1.0.0 schema we had
- Created the new schema incorporating all the new attributes

**Key additions:**

1. **Enhanced `includes` sections** - Added `objectType` field to distinguish teams vs athletes, plus `isCanonical` and `teamId` for athletes

2. **New `gameBreakdown` properties:**
   - **`eventAttributes`** - Complete event metadata (name, dates, sport type, gender, team info, system IDs, venue, lineup)
   - **`breakdownAttributes`** - Breakdown metadata (id, type, priority) 
   - **`hasAthletes`** & **`hasRoster`** - Boolean flags for athlete tagging status

3. **Enhanced `angles`** - Added `startUtc` field for angle timing

4. **Updated schema references** - Changed schema URL examples to reflect v2.0.0

The new schema is fully compatible with the published breakdown structure and includes all the attributes from our specification document. All changes maintain backward compatibility where possible while extending the schema's capabilities.

roman-matviiv commented on schema.json:
```diff
@@ -156,378 +155,606 @@
-            "name": {
+            "id": {
               "type": "string",
-              "description": "Name of the camera angle",
-              "examples": ["HP", "P3", "B1", "HH", "CF", "hd"]
+              "description": "Breakdown ID"
             },
-            "recordedStreamUrl" : {
+            "type": {
               "type": "string",
-              "description": "URL to the recorded video stream",
-              "format": "uri",
-              "examples": ["https://d2s0txx1aqnj4f.cloudfront.net/someTenantName/6736d0dc4c7136f9b31f0269/cloud_hls/0_hd_hls.m3u8"]            
+              "description": "Type of the breakdown",
+              "enum": ["basic", "advanced"]
+            },
+            "priority": {
+              "type": "integer",
+              "description": "Priority of the breakdown (integer number)"
             }
           }
+        },
```
`breakdownAttributes` can be skipped completely if we do not have related data, so we need to specify it as optional. Also, we can be sure that each breakdown has all `id`, `type`, and `priority` attributes at the same time, so it will be good to indicate that each of them can be present or missed individually.

roman-matviiv commented on schema.json:
```diff
@@ -156,378 +155,606 @@
+            "format": "uri",
+            "examples": ["https://d2s0txx1aqnj4f.cloudfront.net/someTenantName/6736d0dc4c7136f9b31f0269/cloud_hls/0_hd_hls.m3u8"]            
+          },
+          "startUtc": {
+            "type": "integer",
+            "description": "Start time of the angle in UTC format",
+            "examples": [1758751698]
+          }
```
It should be a Date as a string, not an integer.

```
"startUtc": {
              "type": "string",
              "format": "date-time",
              "description": "Indicator of event start time in UTC format",
              "examples": ["2025-05-01T12:00:00.000Z"]
            }
```

YambPix commented:
## ✅ Addressed Review Feedback

@roman-matviiv Thank you for the review! I've addressed both of your requested changes:

### 🔧 Changes Made:

**1. Fixed venue field optionality:**
- Issue: Venue ID is not always present (only from MBE event data)
- Fix: Updated description to clarify it's optional for non-external events
- Location: eventAttributes.venue in both schema.json and markdown docs

**2. Fixed startUtc field type in angles:**
- Issue: Was incorrectly defined as integer 
- Fix: Changed to string with date-time format
- Location: angles.startUtc in both schema.json and markdown docs

### 📝 Updated Schema:

venue field - now clearly marked as optional:
```json
"venue": {
  "type": "string",
  "description": "ID of the venue used for the event. Optional field as it can be missed for non-external events",
  "examples": ["60b353ebb59e3af4c8cb2d45"]
}
```

startUtc field - now proper date-time string:
```json
"startUtc": {
  "type": "string",
  "format": "date-time",
  "description": "Indicator of event start time in UTC format",
  "examples": ["2025-05-01T12:00:00.000Z"]
}
```

Both schema.json and the markdown documentation have been updated consistently. Ready for re-review! 🚀
